docs/source/character_set.rst

```diff
@@ -0,0 +1,1230 @@
+    | 20980 | 20980          | EBCDIC, SBCS    | DENMARK, NORWAY                                           | CW      |
+    +-------+----------------+-----------------+-----------------------------------------------------------+---------+
+    | 21314 | 21314          | EBCDIC, DBCS    | KOREAN                                                    | TW      |
+    +-------+----------------+-----------------+-----------------------------------------------------------+---------+
+    | 21317 | 21317          | EBCDIC, DBCS    | SIMPLIFIED CHINESE (S-CH)                                 | TX      |
+    +-------+----------------+-----------------+-----------------------------------------------------------+---------+
+    | 21344 | 21344          | ASCII, SBCS     | ARABIC (WITH MS CONTROLS)                                 | PE      |
+    +-------+----------------+-----------------+-----------------------------------------------------------+---------+
+    | 21427 | 21427          | ASCII, DBCS     | TRADITIONAL CHINESE (T-CH)-IBM BIG-5                      | KE      |
+    +-------+----------------+-----------------+-----------------------------------------------------------+---------+
+    | 21433 | 21433          | ASCII, DBCS     | JAPANESE EUC                                              | S7      |
+    +-------+----------------+-----------------+-----------------------------------------------------------+---------+
+    | 21450 | 21450          | ASCII, MBCS     | KOREAN TCP                                                | LR      |
+    +-------+----------------+-----------------+-----------------------------------------------------------+---------+
+    | 21680 | 21680          | UTF-16          | UCS-2 (VERSION 4.0)                                       | TH      |
+    +-------+----------------+-----------------+-----------------------------------------------------------+---------+
+    | 24613 | 24613          | EBCDIC, SBCS    | INTERNATIONAL                                             | AG      |
+    +-------+----------------+-----------------+-----------------------------------------------------------+---------+
+    | 24876 | 24876          | EBCDIC, DBCS    | JAPAN                                                     | UG      |
+    +-------+----------------+-----------------+-----------------------------------------------------------+---------+
+    | 24877 | 24877          | ASCII, DBCS     | JAPAN PC-DISPLAY                                          | BY      |
+    +-------+----------------+-----------------+-----------------------------------------------------------+---------+
+    | 25013 | 25013          | ASCII, SBCS     | USA PC-DISPLAY                                            | CK      |
+    +-------+----------------+-----------------+-----------------------------------------------------------+---------+
+    | 25076 | 25076          | EBCDIC, SBCS    | DENMARK, NORWAY                                         CX|         |
```

Comment:
The 'ck' appears to be in the wrong column.

Reply:
Nice catch Rich!! 


I went to the docs and looks like it is now showing correctly 

![Screenshot 2025-03-20 at 12 58 26 PM](https://github.com/user-attachments/assets/7fc2c9f4-f5e7-44e4-820a-e5866f433f58)

Reply:
Fixed the table myself